zproject/sentry.py

```diff
@@ -66,6 +66,7 @@ def traces_sampler(sampling_context: dict[str, Any]) -> float | bool:
 def setup_sentry(dsn: str | None, environment: str) -> None:
     from django.conf import settings
 
+    print(f"Sentry, DSN is {dsn}")
```

Comment:
Is this intended to be part of the change?

Reply:
Whoops, failed my self-review.